static/app/components/replays/replayCurrentUrl.tsx

```diff
@@ -29,7 +29,7 @@ function ReplayCurrentUrl() {
 
   if (!replay || !url) {
     return (
-      <TextCopyInput size="sm" disabled>
+      <TextCopyInput aria-label="Current URL" size="sm" disabled>
```

Comment:
Translate here and other labels in the file

Reply:
Good catch! Added